src/main/bash/sdkman-main.sh

```diff
@@ -44,8 +44,21 @@ function sdk {
 			COMMAND="default";;
 		b)
 			COMMAND="broadcast";;
+		ho)
```

Comment:
I'd make this `h` and drop the `help` mnemonic.

Reply:
done